metrics.go

```diff
@@ -0,0 +1,55 @@
+package dht
+
+import (
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+// Copied from https://github.com/census-instrumentation/opencensus-go/blob/master/plugin/ocgrpc/stats_common.go
```

Comment:
Can't these be inferred automatically, or use a default distribution from somewhere? Make it private?

Reply:
They can be made private.